chromedp.go

```diff
@@ -244,7 +244,9 @@ func FromContext(ctx context.Context) *Context {
 // underlying errors happening during cancellation.
 func Cancel(ctx context.Context) error {
 	c := FromContext(ctx)
-	if c == nil {
+	// c.cancel is nil when Cancel is wrongly called with a context returned
+	// by chromedp.NewExecAllocator or chromedp.NewRemoteAllocator.
+	if c == nil || c.cancel == nil {
```

Comment:
Now lets add a test to cover it. Add it after line 263: https://github.com/chromedp/chromedp/blob/97a00b98306467ee7efe556f09fbc9cac0e7c94c/chromedp_test.go#L260-L263

```go
	if err := Cancel(allocCtx); err != ErrInvalidContext {
		t.Fatalf("want error %q, got %q", ErrInvalidContext, err)
	}
```